.github/workflows/package.yaml

```diff
@@ -3,28 +3,23 @@ name: Python package
 on:
   push:
     branches:
-    - master
+      - master
   pull_request:
     branches:
-    - master
+      - master
   workflow_dispatch:
 
 jobs:
 
   build:
-    name: Build for (${{ matrix.python-version }}, ${{ matrix.os }})
-    runs-on: ${{ matrix.os }}
-    strategy:
-      fail-fast: false
-      matrix:
-        os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
-        python-version: ['3.11', '3.12']
+    name: Build for Windows with Python 3.12.9
+    runs-on: windows-latest
```

Comment:
Why so specific and not the matrix or at least more relaxed versions?

Reply:
The results differ slightly depending on the operating system and the Python version. Hence, I test the code using this specific OS and Python version.

Reply:
Okay - something to keep an eye out for in the future.